src/components/hotkeys/hotkeys.service.ts

```diff
@@ -25,8 +25,8 @@ function _getDisplay(combo) {
   const result = [];
 
   for(const k of keys) {
-    if(k === 'mod' && isMac) {
-      result.push(map.command);
+    if(k === 'mod') {
+      result.push(isMac ? map.command : 'ctrl');
```

Comment:
When using `mod` as the key accessor, this should be handled automatically by the Mousetrap library